backend/engine/plugins/trivy_sca/main.py

```diff
@@ -0,0 +1,63 @@
+"""
+trivy SCA plugin
+"""
+import json
+import subprocess
+from engine.plugins.lib import utils
+from engine.plugins.lib.trivy_common.generate_locks import check_package_files
+from engine.plugins.lib.trivy_common.parsing_util import convert_output
+from engine.plugins.lib.trivy_common.parsing_util import parse_output
+
+logger = utils.setup_logging("trivy_sca")
+
+NO_RESULTS_TEXT = "no supported file was detected"
+
+
+def execute_trivy_lock_scan(path: str, include_dev: bool):
+    logger.info(f"Scanning lock-files. Dev-dependencies: {include_dev}")
+    args = ["trivy", "fs", "--include-dev-deps", path, "--format", "json"]
+    if not include_dev:
+        args.remove("--include-dev-deps")
+    proc = subprocess.run(args, capture_output=True, check=False)
+    if proc.returncode != 0:
+        logger.warning(proc.stderr.decode("utf-8"))
+        return None
+    if proc.stdout.decode("utf-8") == "null":
+        logger.warning("No response returned. No files to parse.")
+        return None
+    if proc.stdout and NO_RESULTS_TEXT not in proc.stdout.decode("utf-8"):
+        return proc.stdout.decode("utf-8")
+    logger.error(proc.stderr.decode("utf-8"))
+    return None
+
+
+def main():
+    logger.info("Executing Trivy")
```

Comment:
It will be clear as the logger itself will state trivy_sca and there are other functions it calls that specify it, however, it won't hurt to update that to say trivy_sca for extra clarity